lib/linux/audio.nit

```diff
@@ -0,0 +1,56 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Copyright 2015 Romain Chanoir  <romain.chanoir@viacesi.fr>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Linux audio services
+module audio
+
+import app::audio
+
+# Simple audio asset
+redef class Sound
+
+	# placebo
+	redef fun play do
+		sys.system("aplay ../assets/{name} &")
```

Comment:
You can use the following snippet to hackishly support mp3 files too:

~~~
		if name.has_suffix(".wav") then
			sys.system "aplay -q assets/{name} &"
		else if name.has_suffix(".mp3") then
			sys.system "mpg123 -q assets/{name} &"
		end
~~~

Also note the `-q` option to `aplay` to avoid spamming stdout.